clrenv/lazy_env.py

```diff
@@ -161,6 +162,15 @@ def _get_keyfile_cache():
         _kf_dict_cache = clrypt.read_file_as_dict('keys', 'keys')
     return _kf_dict_cache
 
+def _clear_keyfile_cache():
+    global _kf_dict_cache
+    _kf_dict_cache = {}
```

Comment:
any reason not to just call `.clear()` instead?

Reply:
will do